Makefile

```diff
@@ -68,7 +68,7 @@ oci-image-tool:
 
 schema-fs:
 	@echo "generating schema fs"
-	@cd schema && echo -e "$$(cat ../.header)\n\n$$(go generate)" > fs.go
+	@cd schema && echo "$$(cat ../.header)\n\n$$(go generate)" > fs.go
```

Comment:
If you remove `-e`, you'll get literal `\n`:

    $ echo 'a\n\nb'
    a\n\nb
    $ echo -e 'a\n\nb'
    a

    b

If want POSIX compliance, you probably want to use [printf][1] here.

[1]: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/printf.html